airqo-platform/AirQo-api
# Fix deviceregistry monthsinfront

**_WHAT DOES THIS PR DO?_**
Fix Internal server error showing "monthsInfront is not a function in create-site"

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**
- Checkout to this PR.
- Navigate to `src/device-registry`
- Run command `npm run dev-mac`
- In the console, you shouldn't see the error or warning showing "Accessing non-existent property 'monthsInfront' of module exports inside circular dependency."
- For comparison, you can repeat the steps below on current staging branch where you'll see the error.



**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**
